client/me/purchases/remove-purchase/index.jsx

```diff
@@ -147,9 +148,10 @@ const RemovePurchase = React.createClass( {
 					text: this.state.survey.questionTwoText
 				},
 				'what-better': { text: this.state.survey.questionThreeText },
-				purchase: purchase.productSlug,
 				type: 'cancel'
-			} );
+			};
+
+			survey.addResponses( enrichedSurveyData( surveyData, moment(), selectedSite, purchase ) );
```

Comment:
it doesn't appear like you have introduced this code, but I would like to point out that it's odd to be calling `wpcom` directly from inside a React component and ideally we could refactor that out of here.